app/code/Magento/Captcha/Helper/Adminhtml/Data.php

```diff
@@ -39,19 +47,32 @@ public function __construct(
     /**
      * Returns config value for admin captcha
      *
-     * @param string $key The last part of XML_PATH_$area_CAPTCHA_ constant (case insensitive)
-     * @param \Magento\Store\Model\Store $store
+     * @param  string                     $key   The last part of XML_PATH_$area_CAPTCHA_ constant (case insensitive)
+     * @param  \Magento\Store\Model\Store $store
      * @return \Magento\Framework\App\Config\Element
      */
     public function getConfig($key, $store = null)
     {
         return $this->_backendConfig->getValue('admin/captcha/' . $key);
     }
 
+    /**
+     * Check if reCaptcha is enable for backend
+     *
+     * @return bool
+     */
+    public function reCaptchaEnable()
+    {
+        return (bool)$this->scopeConfig->getValue(
+            self::XML_PATH_ENABLED_BACKEND,
+            ScopeInterface::SCOPE_STORE
```

Comment:
@gauravagarwal1001 I'm not sure if adding Core Bundled Extension's dependency checks in Core modules is a great idea. I will check with the team first before approving this.

Reply:
@gauravagarwal1001 We cannot have CBE dependency in Core modules, do you have any other workaround for this?

Reply:
No, I only added code which depend on configuration of msp reCAPTCHA module.